### PR TITLE
Reformat all the code to match the latest SQ codestyle

### PR DESCRIPTION
**Pull request by ryanhall07**
Most of the noise is coming from the change to indent spaces 2 instead
of the Kotlin standard of 4.

https://github.com/square/java-code-styles/pull/51

This is causing a lot of local diff noise when developing, so figured
one giant change was better.

### CODE REVIEW

**Comment by ryanhall07** [comment]
I only reformatted the KT and java files

```
~/Development/misk rhall-reformat git diff HEAD^1..HEAD --name-only | grep -v ".kt" | grep -v ".java"
~/Development/misk rhall-reformat
```

File: exemplar/src/main/kotlin/com/squareup/exemplar/HelloWebAction.kt [hunk 1]
```diff
@@ -1,29 +1,29 @@
 package com.squareup.exemplar
 
-import misk.web.Get
-import misk.web.PathParam
-import misk.web.QueryParam
-import misk.web.RequestHeaders
-import misk.web.ResponseContentType
+import misk.web.*
```
Comment (tso) [inline exemplar/src/main/kotlin/com/squareup/exemplar/HelloWebAction.kt]: Probably want to not do start imports

Comment (ryanhall07) [reply exemplar/src/main/kotlin/com/squareup/exemplar/HelloWebAction.kt]: hmm that's not good! I'll look into my intellij did this. guess I should actually review this myself :)

Comment (ryanhall07) [reply exemplar/src/main/kotlin/com/squareup/exemplar/HelloWebAction.kt]: Fixed the issue with the sq formatter. Now I just need to get access rights to push my branch :)

Comment (ryanhall07) [reply exemplar/src/main/kotlin/com/squareup/exemplar/HelloWebAction.kt]: https://github.com/square/java-code-styles/pull/52

**Comment by swankjesse** [comment]
I'd prefer to do this as a git history rewrite to avoid breaking `git blame`. It's more work but it avoids having a giant gap in the (short) history of the project.

**Comment by ryanhall07** [comment]
Oh I've never done that before! Are you proposing we reformat each individual commit and amend it? I did a quick google search and couldn't find any tooling to support that

**Comment by swankjesse** [comment]
Here’s a branch with the rewrites:
https://github.com/square/misk/commits/master_reformatted

**Comment by swankjesse** [comment]
And here’s an example of a rewritten commit on [master](https://github.com/square/misk/commit/cdb284d3d3e80a35e16c2155aa5bdae9341fdbbb#diff-f812f946effe075065f618c590a74cf8R1) and [master_reformatted](https://github.com/square/misk/commit/3ae370707acc9d45fd095ee9ebbaec1e72ea8337#diff-f812f946effe075065f618c590a74cf8R1)

**Comment by swankjesse** [comment]
... and the program that recursively “reformats” a directory of Kotlin files. This one just halves the leading spaces on `.kt` files and nothing else.

https://gist.github.com/swankjesse/7952d0c54efc2701fe6fc0bc184549cc

**Comment by swankjesse** [comment]
@mmihic @nbroyles if y’all give me the green light, I’ll run this on `master`. I don’t think I can do it with a pull request though. It’ll just be a push to master with a new rewritten history.

The only hassle is that you’ll need to pull master and fixup in-flight PRs. Possibly by running the gist above.

**Comment by ryanhall07** [comment]
@swankjesse sgtm!

**Comment by stevecstian** [comment]
@swankjesse I'm not 100% sure, but I think you probably want to keep a tag or branch which points to the _`master-before-reformatting`_ to keep old commits reachable, especially for those commits mentioned in GitHub Issues.

**Pull request closed without merging**